stories/core/DatePicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { DatePicker } from 'react-ui-core/src'
+
+export const DefaultDatePicker = (
+  <DatePicker
+    value="10/10/2018"
+    onClick={action('picked a date')}
```

Comment:
I noticed that onClick sends the entire `HtmlInputElement` class to the action logger. Perhaps `onChange` is better because it'll display the `selectedDate` that was passed in `DatePicker.dateChange`?

Reply:
Yeah, this should _really_ be `onChange` in the story...